config/jobs/kubernetes-sigs/sig-windows/release-1.24-windows.yaml

```diff
@@ -30,6 +30,7 @@ periodics:
   extra_refs:
   - org: kubernetes-sigs
     repo: cluster-api-provider-azure
+    # TODO: update to release-1.9 when Go 1.20 is backported to kubekins-e2e.
```

Comment:
1.24 needs to stay on `release-1.6`.
We have problems building the Windows container images in the cloud-provider-azure release-1.24 branch

https://github.com/kubernetes/test-infra/pull/29468

Reply:
It looks like it's already been updated to `release-1.8` in #29248.

Should I remove this `TODO` comment (or all of them)? Or change it do:
`# Don't update to release-1.9`

Reply:
I updated it yesterday to fix the periodic jobs.

test-infra @ master currently has

```yaml
  extra_refs:
  - org: kubernetes-sigs
    repo: cluster-api-provider-azure
    # The Windows 1.24 jobs should stay on CAPZ release-1.6 for now because later versions build CNM and CCM
    # images from k-sigs/cluster-provider-azure repo and the container image builds are not working.
    base_ref: release-1.6
    path_alias: sigs.k8s.io/cluster-api-provider-azure
    workdir: true
```

Reply:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/sig-windows/release-1.24-windows.yaml#L72-L79